test/bigquery.test.ts

```diff
@@ -441,56 +394,23 @@ describe('BigQueryFormatter', () => {
       expect(format(input)).toBe(expected);
     });
 
-    createCmds.snapshotTable.forEach(createSnapshotTable => {
-      it(`Supports ${createSnapshotTable}`, () => {
-        const input = `
-          ${createSnapshotTable} mydataset.mytablesnapshot
-          CLONE mydataset.mytable`;
-        const expected = dedent`
-          ${createSnapshotTable}
-            mydataset.mytablesnapshot CLONE mydataset.mytable`;
-
-        expect(format(input)).toBe(expected);
-      });
-    });
-
-    createCmds.view.concat(createCmds.materializedView).forEach(createView => {
-      it(`Supports ${createView}`, () => {
-        const input = `
-          ${createView} my_dataset.my_view AS (
-          SELECT t1.col1, t1.col2 FROM my_dataset.my_table)`;
-        const expected = dedent`
-          ${createView}
-            my_dataset.my_view AS (
-              SELECT
-                t1.col1,
-                t1.col2
-              FROM
-                my_dataset.my_table
-            )`;
-        expect(format(input)).toBe(expected);
-      });
-    });
-
-    createCmds.externalTable.forEach(createExternalTable => {
-      it(`Supports ${createExternalTable}`, () => {
-        const input = `
-          ${createExternalTable} dataset.CsvTable
-          WITH PARTITION COLUMNS (
-            field_1 STRING,
-            field_2 INT64
-          )
-          OPTIONS(
-            format = 'CSV',
-            uris = ['gs://bucket/path1.csv']
-          )`;
-        const expected = dedent`
-          ${createExternalTable}
-            dataset.CsvTable
-          WITH PARTITION COLUMNS
-            (field_1 STRING, field_2 INT64) OPTIONS(format = 'CSV', uris = ['gs://bucket/path1.csv'])`;
-        expect(format(input)).toBe(expected);
-      });
+    it(`Supports CREATE EXTERNAL TABLE ... WITH PARTITION COLUMN`, () => {
+      const input = `
+        CREATE EXTERNAL TABLE dataset.CsvTable
+        WITH PARTITION COLUMNS (
+          field_1 STRING,
+          field_2 INT64
+        )
+        OPTIONS(
+          format = 'CSV',
+          uris = ['gs://bucket/path1.csv']
+        )`;
+      const expected = dedent`
+        CREATE EXTERNAL TABLE
+          dataset.CsvTable
+        WITH PARTITION COLUMNS
```

Comment:
this output seems worse than the input somehow

Reply:
I agree.

There's a larger problem with our current formatting style, which wants to format everything as:

```
SOME CLAUSE
  content
ANOTHER CLAUSE
  clause content
```

While it often makes more sense to not create an indented block, but instead place things one the same line. For example `SORT BY` and `LIMIT` are more commonly formatted on the same line:

```sql
SORT BY col1 ASC, col2 DESC
LIMIT 10, 8
```

Similarly `CREATE TABLE` is usually formatted as:

```sql
CREATE TABLE customer (
  id INT NOT NULL,
  name VARCHAR NOT NULL,
  address VARCHAR
);
```

But the formatter currently produces:

```sql
CREATE TABLE
  customer (
    id INT NOT NULL,
    name VARCHAR NOT NULL,
    address VARCHAR
  );
```

...which isn't really how anybody in would format it manually.

We'll need to do some rethinking and reorganization of the current formatting approach.